testdata/dnn/download_models.py

```diff
@@ -220,13 +225,13 @@ def get_confirm_token(response):  # in case of large files
         filename='squeezenet_v1.1.caffemodel'),
     Model(
         name='MobileNet-SSD',  # https://github.com/chuanqi305/MobileNet-SSD
-        url='https://drive.google.com/uc?export=download&id=0B3gersZ2cHIxRm5PMWRoTkdHdHc',
-        sha='994d30a8afaa9e754d17d2373b2d62a7dfbaaf7a',
+        url='https://raw.githubusercontent.com/chuanqi305/MobileNet-SSD/97406996b1eee2d40eb0a00ae567cf41e23369f9/mobilenet_iter_73000.caffemodel',
+        sha='19e3ec38842f3e68b02c07a1c24424a1e9db57e9',
         filename='MobileNetSSD_deploy.caffemodel'),
```

Comment:
We should not change content (check sum) under the same name.

This is a limitation of shared network storage. There is no versioning.

Reply:
I will try to fix that. Hopefuly, most of the ONNX models have not changed - only their input/output data (propably, I could find a workaround). Archives might be renamed.